### PR TITLE
CDS-45146 Azure Artifacts added

### PR DESCRIPTION
**Pull request by michaelcretzman**
https://harness.atlassian.net/browse/CDS-45146

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ x] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* https://harness.atlassian.net/browse/CDS-45146

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/continuous-delivery/x-platform-cd-features/services/artifact-sources.md [hunk 1]
```diff
@@ -922,6 +922,196 @@ Ensure the Harness delegate you have installed can reach your Google Artifact Re
...
+
+<details>
+<summary>Use Azure Artifacts</summary>
+
+You connect to your Azure DevOps artifacts using a Harness Azure Artifacts connector.
+
+```mdx-code-block
+import Tabs15 from '@theme/Tabs';
+import TabItem15 from '@theme/TabItem';
+```
```
Comment (aimurphy) [inline docs/continuous-delivery/x-platform-cd-features/services/artifact-sources.md:935]: It makes you add numbers every time you include the "import" lines. If you just put one set of import lines at the top of the file, you dont need to include it every time you want to add a set of tabs and you wont need the numbers.

File: docs/continuous-delivery/x-platform-cd-features/services/artifact-sources.md [hunk 2]
```diff
@@ -922,6 +922,196 @@ Ensure the Harness delegate you have installed can reach your Google Artifact Re
...
+
+```mdx-code-block
+  </TabItem15>
+  <TabItem15 value="Pipeline Studio" label="Pipeline Studio">
+```
+
+<details>
+<summary>Azure Artifact connector</summary>
+
+1. In you Harness project, in **Connectors**, select **New Connector**, and then select **Azure Artifacts**.
```
Comment (aimurphy) [inline docs/continuous-delivery/x-platform-cd-features/services/artifact-sources.md:1043]:
```suggestion
1. In your Harness project, in **Connectors**, select **New Connector**, and then select **Azure Artifacts**.
```

File: docs/continuous-delivery/x-platform-cd-features/services/artifact-sources.md [hunk 3]
```diff
@@ -922,6 +922,196 @@ Ensure the Harness delegate you have installed can reach your Google Artifact Re
...
+6. Save the connector.
+
+</details>
+
+
+<details>
+<summary>Add an Azure Artifact artifact</summary>
+
+1. In a Harness service, select **Configuration**.
+2. In **Deployment Type**, select one of the supported deployment types (deployment type support is described below).
```
Comment (aimurphy) [inline docs/continuous-delivery/x-platform-cd-features/services/artifact-sources.md:1057]:
```suggestion
2. In **Deployment Type**, select one of the [supported deployment types](#deployment-type-support).
```

Not sure of the exact syntax for the anchor.

File: docs/continuous-delivery/x-platform-cd-features/services/artifact-sources.md [hunk 4]
```diff
@@ -922,6 +922,196 @@ Ensure the Harness delegate you have installed can reach your Google Artifact Re
...
+3. In **Artifacts**, select **Add Artifact Source**.
+4. In **Specify Artifact Repository Type**, select **Azure Artifacts**, and select **Continue**.
+5. In **Azure Artifacts Repository**, select or create an Azure Artifacts connector that connects to your Azure DevOps organization, and then select **Continue**.
+6. In **Artifact Details**, enter the following:
+   1. **Artifact Source Identifier:** enter the name for the artifact in Harness.
+   2. **Scope:** select **Org** or **Project**.
+   3. **Feed:** select the artifact feed.
+   4. **Package Name:** select the name of the package on Azure Artifacts.
+   5. **Version:** select the artifact version to use or set the option as a runtime input or expression.
```
Comment (aimurphy) [inline docs/continuous-delivery/x-platform-cd-features/services/artifact-sources.md:1066]:
```suggestion
   1. **Artifact Source Identifier:** Enter the name for the artifact in Harness.
   2. **Scope:** Select **Org** or **Project**.
   3. **Feed:** Select the artifact feed.
   4. **Package Name:** Select the name of the package on Azure Artifacts.
   5. **Version:** Select the artifact version to use or set the option as a runtime input or expression.
```

**Review by aimurphy — APPROVED** [review]
approved with some comments

File: docs/continuous-delivery/x-platform-cd-features/services/artifact-sources.md (hunk 1 above)
Comment (michaelcretzman) [reply docs/continuous-delivery/x-platform-cd-features/services/artifact-sources.md:935]: thanks

**Review by vtxorxwitty — APPROVED** [review]

**Pull request merged**